perf/src/recycler.rs

```diff
@@ -148,9 +153,12 @@ impl<T: Default + Reset> RecyclerX<T> {
             if gc.len() > RECYCLER_SHRINK_SIZE
                 && self.size_factor.load(Ordering::Acquire) >= SIZE_FACTOR_AFTER_SHRINK
             {
+                let mut num_recycled: usize = 0;
                 for mut x in gc.drain(RECYCLER_SHRINK_SIZE..) {
                     x.set_recycler(Weak::default());
+                    num_recycled = num_recycled.saturating_add(1);
                 }
+                self.stats.freed.fetch_add(num_recycled, Ordering::Relaxed);
```

Comment:
can we just calculate this directly instead of incrementing inside the loop?
I think it will be like
```rust
self.stats.freed.fetch_add(gc.len() - RECYCLER_SHRINK_SIZE, Ordering::Relaxed);
```
before the for loop.

Reply:
Fixed.